### PR TITLE
Link generation is done by HtmlHelper::link() added escape and escapeTit...

### PR DESCRIPTION
**Pull request by redd**
...le options.

### CODE REVIEW

**Comment by torifat** [comment]
Thanks for the pull request but the test is failing.

**Comment by redd** [comment]
Fixed

**Comment by torifat** [comment]
Awesome :+1: , but do we need both `alt` & `title`?

**Comment by redd** [comment]
The title was duplicated on `` so I've removed it. Title stays on `<a>` element as it was earlier, I didn't touch it.

**Comment by redd** [comment]
Do you need anything else?

**Comment by torifat** [comment]
@redd No, I just wanted to have a look at it. But, Sorry I was very busy.

File: Test/Case/View/Helper/MenuBuilderHelperTest.php [hunk 1]
```diff
@@ -1102,7 +1102,7 @@ public function testImageMenu() {
 			'</li',
 			'<li',
 			array('a' => array('title' => 'Item 2', 'href' => '/item-2')),
-			array('img' => array('src' => '/path/my-image.jpg', 'alt' => 'Item 2', 'title' => 'Item 2')),
```
Comment (torifat) [inline Test/Case/View/Helper/MenuBuilderHelperTest.php:1105]: @redd I was little bit concerned about this one. This means the change will break the current versions. Right?

Comment (redd) [inline Test/Case/View/Helper/MenuBuilderHelperTest.php:1105]: Yes, but `title` is reduntant when you've `alt` on image and `title` on the link.

Comment (torifat) [inline Test/Case/View/Helper/MenuBuilderHelperTest.php:1105]: Yes, I agree with you that it is redundant. But, before pushing this change I have to add a how to upgrade section. Or, many existing users will end up having a broken site.

Comment (redd) [inline Test/Case/View/Helper/MenuBuilderHelperTest.php:1105]: I think that (eventually) don't showing `title` is a little abuse to have broken sites :) But when image is in the link it should without problems show link's `title`.

Comment (torifat) [inline Test/Case/View/Helper/MenuBuilderHelperTest.php:1105]: :+1: Sorry, I forgot what it is :smile:

**Pull request merged**